changelog

```diff
@@ -1,3 +1,400 @@
+Version 0.10.0
+  * Breaking changes:
+    * The accompany objective on mission npcs no longer implicitly sets the save objective. Plugin missions that had "accompany" but not "save" should be changed to include "save" if the intention of the mission is that the npcs live (without the bug this time). (@Amazinite, @RisingLeaf)
+  * Big changes:
+    # Content
+    * Hai Reveal! (@MasterOfGrey)
+      * Various fixes to missions. (@bene-dictator, @Hurleveur, @McloughlinGuy, @RisingLeaf, @UnorderedSigh)
+      * Various typo fixes. (@bene-dictator, @Hatrask, @McloughlinGuy, @UnorderedSigh, @williaji)
+      * Relocate the new "Deep Space" systems. (@Quantumshark)
+      * Various fixes to other content. (@TomGoodIdea)
+    * A new alien faction: The Gegno. (@Saugia)
+      * TODO: what is included here.
+      * Improve some wording. (@Zitchas)
```

Comment:
I know the TODO is to remind you, but is the line about Zitchas specifically pertaining to the Gegno since it's lower in the hierarcy, or is it supposed to be separate?